src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/MitochondrialCallerArgumentCollection.java

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.hellbender.tools.walkers.mutect;
+
+import org.broadinstitute.barclay.argparser.Advanced;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.tools.walkers.haplotypecaller.AssemblyBasedCallerArgumentCollection;
+
+public class MitochondrialCallerArgumentCollection extends AssemblyBasedCallerArgumentCollection {
```

Comment:
@ldgauthier Longer term I could change the Mutect docs to be more general.

Reply:
There is no way to override `@Argument` in a subclass, but @meganshand with some refactoring you could define an interface or base class with abstract methods that return values of interest, and then dynamically select the concrete implementation on a per-tool basis. GATK and Picard both use this to override doc strings or required attributes in command line args. See [this](https://github.com/broadinstitute/picard/blob/b02e42e0ca82efae70ee572766cbe101b6a2305e/src/main/java/picard/cmdline/CommandLineProgram.java#L120) and [this](https://github.com/broadinstitute/picard/blob/b02e42e0ca82efae70ee572766cbe101b6a2305e/src/main/java/picard/cmdline/CommandLineProgram.java#L182) for example. 

`tumorSample` would require special handling because for M2 its provided in the CLI and for Mito its derived from the input header.